flink-streaming-java/src/main/java/org/apache/flink/streaming/api/graph/StreamEdge.java

```diff
@@ -130,7 +122,7 @@ public boolean equals(Object o) {
 
 	@Override
 	public String toString() {
-		return "(" + sourceVertex + " -> " + targetVertex + ", typeNumber=" + typeNumber
+		return "(" + sourceId + " -> " + targetId + ", typeNumber=" + typeNumber
```

Comment:
I recommend keeping the “operatorName” information so it looks more intuitive. e.g.: 
`sourceId` -> `operatorName + "-" + sourceId`
So, maybe we need add `sourceOperatorName` and `targetOperatorName`, What do you think?

Reply:
I think name is very important for debug, and more debug info is necessary , What do you think? @tillrohrmann @aljoscha

Reply:
I discussed with @sunjincheng121 and decided to add _sourceOperatorName_ and _targetOperatorName_.